### ISSUE TITLE
SitePrism::NoUrlMatcherForPage following documentation example

### ISSUE THREAD

**Issue by conzett** [natritmeyer/site_prism#101]
The documentation shows the following example:

``` ruby
class Account < SitePrism::Page
  set_url "/accounts/{id}{?query*}"
end
```

It goes onto say that the following test would pass:

``` ruby
@account_page = Account.new
@account_page.load(id: 22, query: { token: "ca2786616a4285bc" })

expect(@account_page.current_url).to end_with "/accounts/22?token=ca2786616a4285bc"
expect(@account_page).to be_displayed
```

I have a very similar setup I'm trying to test, and I am getting the following error:

``` shell
Failure/Error: expect(index_page).to be_displayed
SitePrism::NoUrlMatcherForPage:
  SitePrism::NoUrlMatcherForPage
# /Users/gconzett/.gem/ruby/2.0.0/gems/site_prism-2.6/lib/site_prism/page.rb:14:in `displayed?'
```

The page object is as follows:

``` ruby
require_relative '../sections/widget_section.rb'          
require_relative '../sections/widget_fields_section.rb'   

class WidgetIndexPage < SitePrism::Page                    
  set_url '/widgets'                                      

  section :fields, WidgetFieldsSection, 'form'             
  sections :widgets, WidgetSection, 'tbody tr'    

  element :create_widget_button, '#create-widget'
end
```

And the spec itself has this simple assertion which fails:

``` ruby
feature 'Creating a widget' do      
  scenario 'with valid input' do             
    index_page = WidgetIndexPage.new 
    index_page.load                          
    expect(index_page).to be_displayed
  end
end
```

Unfortunately this leads to the above mentioned error. Do I have to define some kind of regex matcher? The coded leads me to believe that it should just use the string for the URL. Any info would be appreciated. Thanks!

**Comment by adamalbrecht** [comment]
+1 I'm having the same issue

**Comment by sherbhachu** [comment]
+1 I have this too, running 2.2.0.

The page I am visiting is...

http://my.website.url/product

and the page object has...

set_url "#{my.website.url}/product"

**Comment by natritmeyer** [comment]
Ah, this is happening because the `README.md` as shown on github reflects a couple of pull requests that have been merged in; but I haven't pushed an updated version of the site_prism gem :/ I'll be doing that over the next few days :)

**Comment by JohnSmall** [comment]
Ah, when is the new gem going to be released? I just updated to version 2.6 from 2.2 and it didn't fix the problem. Should we take the latest commit from here and use that?

**Comment by vmpj** [comment]
Well there you go. 2.7 fixed this issue for me. Thx Nat! This is a great gem!
